config/defines_uri.inc.php

```diff
@@ -32,7 +32,7 @@
 define('_THEME_IMG_DIR_', _THEME_DIR_.'assets/img/');
 define('_THEME_CSS_DIR_', _THEME_DIR_.'assets/css/');
 define('_THEME_JS_DIR_', _THEME_DIR_.'assets/js/');
-define('_PS_THEME_OVERRIDE_DIR_', _PS_THEME_DIR_.'override/');
+define('_PS_THEME_OVERRIDE_DIR_', _PS_THEME_DIR_.'templates/override/');
```

Comment:
I think we can remove this since we're not using it at all and we don't need backwards compatibility with themes.